classes/privacy/provider.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace block_equella_tasks\privacy;
+
+class provider implements \core_privacy\local\metadata\null_provider {
+    // this plugin does not store any personal user data.
+    /**
+     * Get the language string identifier with the component's language
+     * file to explain why this plugin stores no data.
+     */
+    public static function get_reason(): string {
+        return 'privacy:metadata';
+    }
+}
```

Comment:
I'm going to say, if you're opening your file with the php tag (`<?php`) then you should probably end the file closing it (`?>`) to be sure.

Reply:
PSR-2, the PHP code style recommends never adding `?>` to pure PHP files.
https://www.php-fig.org/psr/psr-2/#22-files

Reply:
Ah! Okay, cool. If that's what the PHP world likes, I'm happy to go with that. :grin: